dyn_em/module_diffusion_em.F

```diff
@@ -1637,7 +1637,7 @@ SUBROUTINE calculate_N2( config_flags, BN2, moist,           &
       ELSE
         BN2(i,k,j) = g * ( (theta(i,k+1,j) - theta(i,k-1,j) ) /  &
                      theta(i,k,j) / tmpdz +  &
-                     1.61 * ( moist(i,k+1,j,P_QV) - moist(i,k-1,j,P_QV) ) / &
+                     0.61 * ( moist(i,k+1,j,P_QV) - moist(i,k-1,j,P_QV) ) / &
```

Comment:
@dudhia @weiwangncar 
Jimy,
I agree with Wei that these three N^2 corrections are separate from the vertical diffusion fix.